tests/tests_pytorch/loops/epoch/test_training_epoch_loop.py

```diff
@@ -265,3 +266,33 @@ def test_no_val_on_train_epoch_loop_restart(tmpdir):
     ) as advance_mocked:
         trainer.fit(model, ckpt_path=ckpt_path)
         assert advance_mocked.call_count == 1
+
+
+@pytest.mark.parametrize(
+    "min_epochs, min_steps, current_epoch, global_step, epoch_loop_done, raise_info_msg",
+    [
+        (None, None, 1, 4, True, False),
+        (4, None, 1, 4, False, True),
+        (4, 2, 1, 4, False, True),
+        (4, None, 1, 10, True, False),
+        (4, 3, 1, 3, False, True),
+        (4, 10, 1, 10, True, False),
+        (None, 4, 1, 4, False, False),
+        (None, 10, 1, 10, True, False),
+    ],
+)
+def test_should_stop_early_stopping_conditions(
```

Comment:
What is the difference to the test in the other file?

Reply:
the message and conditions of when they would come up respectively.
This one is about epoch loop, which says that the min_values are not met, and training will continue.

the other one stops the training since conditions are met.

Reply:
let me add descriptions and update test name to make it more clear